core/src/mender-utils.c

```diff
@@ -653,3 +653,55 @@ mender_utils_compare_wildcard(const char *str, const char *wildcard_str, bool *m
 
     return MENDER_OK;
 }
+
+bool
+mender_utils_validate_deployment_data(const cJSON *deployment_data, int version) {
```

Comment:
I'd use an `enum` for `version`.

Reply:
I changed it to `double` since it is now hidden behind an API and I wanted to avoid type casting. Do you still think this should be `enum`?